tmt/steps/report/reportportal.py

```diff
@@ -0,0 +1,89 @@
+import dataclasses
+import io
+import os
+import zipfile
+from typing import List, Optional
+
+import click
+import junit_xml
+
+import tmt.steps.report
+import tmt.utils
+
+from . import junit
+
+
+@dataclasses.dataclass
+class ReportPortalData(tmt.steps.report.ReportStepData):
```

Comment:
Could you please add the new classes to the [class overview](https://github.com/teemtee/tmt/blob/main/docs/classes.rst#phases) as well?